extensions/amp-timeago/0.1/amp-timeago.js

```diff
@@ -57,8 +57,8 @@ export class AmpTimeAgo extends AMP.BaseElement {
       this.win.document.documentElement.lang;
     this.title_ = this.element.textContent.trim();
 
-    this.element.title = this.title_;
     this.element.textContent = '';
+    this.timeElement_.setAttribute('role', 'text');
```

Comment:
Are you intentionally setting the `role` of the `time` element instead of the root `amp-timeago` element?

I believe `<amp-timeago role="text" ...>...</amp-timeago>` in the example results in the `amp-timeago` element having `role="text"` rather than the `time` element. I don't know if setting `role="text"` on the `time` element has the same a11y result

Reply:
You're absolutely right, I submitted this PR too soon and did the wrong thing.

Fixing up.

Reply:
Awesome! Thanks for getting this fix together so quickly :pray: